devices/GnssDevice/Nmea0183Parser.cs

```diff
@@ -0,0 +1,114 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.Common.GnssDevice
+{
+    /// <summary>
+    /// Provides methods for parsing NMEA0183 data from a Gnss device.
+    /// </summary>
+    public static class Nmea0183Parser
+    {
+        /// <summary>
+        /// Gets or sets a list of NMEA0183 data objects.
+        /// </summary>
+        public static INmeaData[] MneaDatas { get; set; }
```

Comment:
How this will be populated?

Reply:
I just realized I missed a push :-( All is here now